### ISSUE TITLE
i2c_2_clock_freq_overlay.dtbo not part of olinuxino-overlay

### ISSUE THREAD

**Issue by markus2330** [ElektraInitiative/opensesame#17]
if `i2c_2_clock_freq_overlay.dtbo` is present in `/usr/lib/olinuxino-overlays/sun7i-a20` you get following error:

```
sudo olinuxino-overlay
`Error at 'compatible': FDT_ERR_NOTFOUND`
```
and `i2c_2_clock_freq_overlay.dtbo` cannot be selected in the list.

**Comment by fel115** [comment]
I added the compatible and description attribute to the devicetree overlay, should I also add the sourcecode of the devicetree overlay to the project? If so in which folder should I add the sourcecode file?

**Comment by markus2330** [comment]
Good point, yes of course, the source code should be added.

> If so in which folder should I add the sourcecode file?

Depends on number of files/size. If it is only one file, probably directly next to the binary file.

**Comment by markus2330** [comment]
It is still not listed in `sudo olinuxino-overlay`, probably it filters based on the name (but I don't know).

**Comment by fel115** [comment]
I analysed the `olinuxino-overlay`-script and I think the problem is, because I added in the following line a whitespace between the comma and sun7i-a20. 
Changing this `compatible = "allwinner, sun7i-a20";` to `compatible = "allwinner,sun7i-a20";` cloud solve this issue.

**Comment by fel115** [comment]
This issue is closed with the PR #29
